src/components/molecules/PlanRoom/RoomTables/Table/index.tsx

```diff
@@ -175,4 +246,8 @@ export const SaveTables = () => {
   SaveBoard(Save());
 };
 
+export const DeleteTables = () => {
```

Comment:
Actually I don't see a point in making a function that only calls another function.
You can call `ClearBoard` directly.

Reply:
I know it should look different the way I did it, unfortunately I was quick to find workarounds for things I didn't quite know how to do. The ClearBoeard function I could not export from this file to be available externally. Unfortunately there were errors that I did not understand. Wasting no time I created solutions that were also not quite in line with my initial ideas for writing it. Personally I do not like the content of this whole file. I would write it differently. However, the shortcomings from Reackt were there. Unfortunately, there is nothing I can do about it. I wanted to leave it this way so that I could understand, in further possible presentations


![image](https://user-images.githubusercontent.com/43721352/123558416-6a18ab80-d796-11eb-83fb-fddafb0d8a68.png)

![image](https://user-images.githubusercontent.com/43721352/123558378-2aea5a80-d796-11eb-85c6-c9db08314c77.png)